pkg/core/native/native_test/notary_test.go

```diff
@@ -26,6 +26,9 @@ import (
 func newNotaryClient(t *testing.T) *neotest.ContractInvoker {
 	bc, acc := chain.NewSingleWithCustomConfig(t, func(cfg *config.Blockchain) {
 		cfg.P2PSigExtensions = true
+		cfg.Hardforks = map[string]uint32{
+			config.HFEchidna.String(): 0,
```

Comment:
Likely not needed is we're to declare Echidna as stable one.

Reply:
I'm done with rebases. Making Echidna a stable fork is a part of https://github.com/nspcc-dev/neo-go/pull/3851, and for now we need this to make tests work properly.